### ISSUE TITLE
bug with long table

### ISSUE THREAD

**Issue by geoffreymcgill** [retypeapp/retype#192]
### Discussed in https://github.com/retypeapp/retype/discussions/191

<div type='discussions-op-text'>

<sup>Originally posted by **febreze21** November  4, 2021</sup>
That appears when i enter:

```
Techniciens | commun | Administratifs | commun | Techniciens de Chassignieu | Administratifs de Chassignieu | Direction | Profils | utilisateurs | Dossier de base
--- | --- | --- | --- | --- | --- | ---
1Go | 500Mo | 1Go | 500Mo | 2Go | 4Go | 4Go 
```

![6f8HshFQ8p](https://user-images.githubusercontent.com/61574808/140290795-bcc1b9d9-c243-494a-a77c-48029439794a.png)

Maybe you could change this with a slider like the code block ?
</div>

**Comment by geoffreymcgill** [comment]
Thanks for letting us know about this rendering issue. I believe there is an easy fix and we will get this into `v1.12`.

If (when) you upgrade to `v1.12`, the table rendering should be working better for you. No changes within your `.md` content will be required.

**Comment by geoffreymcgill** [comment]
I think we have this issue fixed and will include in the `v1.12` release. Still some work to do and final testing, but things are looking good.

**Comment by geoffreymcgill** [comment]
This issue was fixed and released today in Retype `v1.11.1`.

Hope this helps.
